arangod/WasmServer/WasmServerFeature.cpp

```diff
@@ -68,6 +68,7 @@ auto WasmServerFeature::executeFunction(std::string const& name, uint64_t a,
     return function.call<uint64_t>(a, b);
```

Comment:
What happens if the function does not exist in the runtime?

Reply:
I modified the Wasm3cpp.h file such that now `runtime.find_function(name.c_str())` will return a result instead of throwing an error. Then `call` is only called on a valid function.